config/config.go

```diff
@@ -24,10 +23,10 @@ var (
 func DefaultConfig() *cctypes.Config {
 	return &cctypes.Config{
 		AppStore: &cctypes.AppStoreConfig{
-			NumEvmKeysToPrune: 50,
+			NumEvmKeysToPrune: 0,
 		},
 		Evm: &cctypes.EvmConfig{
-			GasLimit: math.MaxUint64,
+			GasLimit: 0,
```

Comment:
Seeing as this is the default value for the data type what's the point of `DefaultConfig()`?

Reply:
This is for iniitializing `Evm` and `AppStore` config or it will be nil.

Reply:
You can probably get accomplish the same thing by using the getter functions on the generated protobuf, they appear to deal with `nil`.

Reply:
you cannot just pass uninitialised config like
```
var cfg3 cctypes.Config
SetConfigSetting(&cfg3, "AppStore.NumEvmKeysToPrune", "1000") 
```
We need a function that initialise the config, allocating memory for `SetConfigSetting` to set value to the memory.
https://github.com/loomnetwork/go-loom/blob/cdb4849c08fc2bf805b2fdcc7a6d3ac65c9affe7/config/config.go#L38